sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/HierarchiesSamples.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.IoT.TimeSeriesInsights.Models;
+using static Azure.IoT.TimeSeriesInsights.Samples.SampleLogger;
+
+namespace Azure.IoT.TimeSeriesInsights.Samples
+{
+    internal class HierarchiesSamples
+    {
+        /// <summary>
+        /// This sample demonstrates usage of Time Series Insights hierarchy APIs.
+        /// </summary>
+        public async Task RunSamplesAsync(TimeSeriesInsightsClient client)
+        {
+            // For the purpose of keeping code snippets readable to the user, hardcoded string literals are used in place of assigned variables, eg Ids.
+            // Despite not being a good code practice, this prevents code snippets from being out of context for the user when making API calls that accept Ids as parameters.
+
+            PrintHeader("TIME SERIES INSIGHTS HIERARCHIES SAMPLE");
+
+            #region Snippet:TimeSeriesInsightsSampleCreateHierarchies
+            var tsiHierarchyName = "sampleHierarchy";
+            var tsiInstanceField1 = "hiearchyLevel1";
```

Comment:
I admit, hierarchy is a tricky word to spell :)

Reply:
I am sure you had to review after you typed as well 😆